### PR TITLE
Updated supported Django versions

### PR DESCRIPTION
**Pull request by smithdc1**
Django 3.0 is now unsupported. Django 4.0 is still in alpha, release date is January 2022.

Source: https://www.djangoproject.com/download/

### CODE REVIEW

File: tests/testproject/settings.py [hunk 1]
```diff
@@ -29,3 +29,5 @@
         'APP_DIRS': True,
     },
 ]
+
+USE_TZ = False
```
Comment (smithdc1) [inline tests/testproject/settings.py:33]: This should probably be `True` but would need more work on the tests. As that's not particularly important to this package I'm not sure it's a concern?

File: tests/unit/test_utils.py [hunk 2]
```diff
@@ -37,7 +37,7 @@ class MyUser(AbstractBaseUser):
     USERNAME_FIELD = 'identity'
 
     class Meta:
-        abstract = True
+        abstract = False
```
Comment (smithdc1) [inline tests/unit/test_utils.py:40]: I'm not sure about this change, again maybe it is ok here but it seems like a breaking change in Django and I can't see it in the release notes.

File: tests/testproject/settings.py (hunk 1 above)
Comment (smithdc1) [reply tests/testproject/settings.py:33]: It was a 3.2 change, need to add support for that as well.

Comment (bittner) [reply tests/testproject/settings.py:33]: Not sure if that's really unimportant. OTOH, we don't handle dates on the server side with analytical, do we? Is any such logic all in the JavaScript code of the services, maybe?  Can we think of any tests the might make sense?

File: tox.ini [hunk 3]
```diff
@@ -10,8 +11,9 @@ deps =
     coverage
     pytest-django
     django22: Django>=2.2,<3.0
-    django30: Django>=3.0,<3.1
     django31: Django>=3.1,<3.2
+    django32: Django>=3.2,<4.0
+    django40: Django>=4.0a1,<4.2
```
Comment (bittner) [inline tox.ini:16]: Shouldn't that be `<4.1`?

**Review by bittner — CHANGES_REQUESTED** [review]

**Comment by bittner** [comment]
BTW, @smithdc1 would it make sense for you to become co-maintainer, maybe?

File: tox.ini (hunk 3 above)
Comment (smithdc1) [reply tox.ini:16]: Yes. 🤦‍♂️

**Review by bittner — APPROVED** [review]

**Comment by smithdc1** [comment]
Thanks for merging this. 👍

> BTW, @smithdc1 would it make sense for you to become co-maintainer, maybe? 

I'm interested. Are there notes/docs somewhere on what this means?

**Comment by bittner** [comment]
> I'm interested. Are there notes/docs somewhere on what this means?

That simply means you get more work. :smirk:  Less sleep & more responsibility. :dizzy_face: :1st_place_medal: 

@jezdez Is there anything specific that needs to be done to make @smithdc1 a (co-) maintainer of django-analytical?

**Comment by bittner** [comment]
@smithdc1 Can you please open a [project lead issue](https://github.com/jazzband/help/issues/new/choose) as described [in the Jazzband docs](https://jazzband.co/about/releases#security)?

**Comment by bittner** [comment]
> @smithdc1 Can you please open a [project lead issue](https://github.com/jazzband/help/issues/new/choose) as described [in the Jazzband docs](https://jazzband.co/about/releases#security), asking to become [a project lead](https://jazzband.co/projects/django-analytical) for django-analytical?

@smithdc1 Please. Thank you!

**Pull request merged**